rust/parquet/src/encodings/rle.rs

```diff
@@ -830,7 +826,7 @@ mod tests {
             values.clear();
             let mut rng = thread_rng();
             let seed_vec: Vec<u8> =
-                Standard.sample_iter(&mut rng).take(seed_len).collect();
+                rng.sample_iter::<u8, _>(&Standard).take(seed_len).collect();
```

Comment:
This is an example of type inferences that would become compiler errors with the inclusion of the `prettytable-rs` crate. I have fixed this code, but the original error being generated was:

```
error[E0282]: type annotations needed
   --> parquet/src/encodings/rle.rs:833:26
    |
833 |                 Standard.sample_iter(&mut rng).take(seed_len).collect();
    |                          ^^^^^^^^^^^ cannot infer type for `T`
```